Kernel/Storage/StorageManagement.cpp

```diff
@@ -286,6 +288,13 @@ UNMAP_AFTER_INIT void StorageManagement::determine_nvme_boot_device()
     });
 }
 
+UNMAP_AFTER_INIT void StorageManagement::determine_ramdisk_boot_device()
```

Comment:
Please put a big FIXME here to say this is temporary until the initramfs patches are in.

Reply:
Sure, will do that!